apps/public/.env.docker

```
@@ -0,0 +1,5 @@
+DATABASE_HOST=mysql
+DATABASE_PORT=3306
+DATABASE_USER=root
+DATABASE_PASSWORD=sverige8
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_PASSWORD key should go before the DATABASE_PORT key

(UnorderedKey)